### PR TITLE
Create are_all_terminal_types_four_pipe.md

### PR DESCRIPTION
**Pull request by claperle**

### CODE REVIEW

File: docs/ruleset_functions/is_terminal_type_four_pipe(terminal.id).md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+- **terminal_b.id**: The id of the terminal unit to evaluate.  
+
+**Returns:**  
+- **is_terminal_type_four_pipe**: The function returns TRUE if the terminal unit type is FOUR_PIPE_FAN_COIL_UNIT. It returns FALSE if the terminal unit is a type other than FOUR_PIPE_FAN_COIL_UNIT.    
+ 
+**Function Call:** None  
+
+## Logic: 
+- Create an object for the terminal unit: `terminal_b = terminal_b.id`  
+- Check if the terminal unit type is FOUR_PIPE_FAN_COIL_UNIT: `if terminal_b.type == "FOUR_PIPE_FAN_COIL_UNIT": is_terminal_type_four_pipe = TRUE`
```
Comment (mkarpman) [inline docs/ruleset_functions/is_terminal_type_four_pipe(terminal.id).md:16]: Should we additionally check that terminal has cooling source = chilled water loop and heating source = hot water loop? That would also affect the check for Systems 1a and 1c. 

Also, consider looping through all terminal units served by a given HVAC system.

Comment (claperle) [reply docs/ruleset_functions/is_terminal_type_four_pipe(terminal.id).md]: I add this to system_1 function logic

Comment (supriyagoel) [reply docs/ruleset_functions/is_terminal_type_four_pipe(terminal.id).md]: @mkarpman and @claperle I think it is correct to have the HW and CHW defined at the HVAC system level and not the terminal level. This is the way we do it for TSPR as well
If you think the correct answer is terminal, then what should the heating and cooling source at HVAC system level be?

**Review by supriyagoel — COMMENTED** [review]
function looks good to me, I'll wait for @mkarpman 's response to my comment before approving.

File: docs/ruleset_functions/is_terminal_type_four_pipe(terminal.id).md (hunk 1 above)
Comment (claperle) [reply docs/ruleset_functions/is_terminal_type_four_pipe(terminal.id).md]: @supriyagoel  The answer to your question is yes. The master runs a function that get all of the terminals associated with the hvac system and then passes the list to the terminal related functions.

File: docs/ruleset_functions/are_all_terminal_types_four_pipe(hvac.id).md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# are_all_terminal_types_four_pipe
```
Comment (claperle) [inline docs/ruleset_functions/are_all_terminal_types_four_pipe(hvac.id).md:1]: Function related to how terminal type is defined.

Comment (mkarpman) [reply docs/ruleset_functions/are_all_terminal_types_four_pipe(hvac.id).md:1]: Based on the latest decisions this function is obsolete.

**Pull request closed without merging**